lib/coffeescript/views/text.coffee

```diff
@@ -66,7 +66,9 @@ export default class TextView extends View
         # Text shadow.
         textShadow = @getTextShadow()
```

Comment:
What about this line?

Reply:
I think we still need to check for the old text_shadow to make sure this doesn't break backwards, so I left in the old way and use it as a fallback.